test/util.js

```diff
@@ -60,16 +58,13 @@ function createSubsHelper({ provider }) {
 }
 
 function createSubGenerator({ subType, provider }) {
```

Comment:
we can just remove pify and replace with
```suggestion
function createSubGenerator({ subType, provider }) {
  return async function () {
    const args = [].slice.call(arguments);
    args.unshift(subType);
    const id = await provider.request({
      method: "eth_subscribe",
      params: args,
    });
    return createNewSub({ id, provider });
  };
```